test/tools/ossfuzz/protoToSol.cpp

```diff
@@ -0,0 +1,228 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <test/tools/ossfuzz/protoToSol.h>
+
+#include <liblangutil/Exceptions.h>
+
+#include <libsolutil/Whiskers.h>
+
+#include <sstream>
+
+using namespace solidity::test::solprotofuzzer;
+using namespace solidity::util;
+using namespace std;
+
+string ProtoConverter::protoToSolidity(Program const& _p)
+{
+	// Create random number generator with fuzzer supplied
+	// seed.
+	m_randomGen = make_shared<SolRandomNumGenerator>(_p.seed());
+	return visit(_p);
+}
+
+pair<string, string> ProtoConverter::generateTestCase(TestContract const& _testContract)
+{
+	ostringstream testCode;
+	string usingLibDecl;
+	switch (_testContract.type())
+	{
+	case TestContract::LIBRARY:
+	{
+		m_libraryTest = true;
+		auto testTuple = pseudoRandomLibraryTest();
+		m_libraryName = get<0>(testTuple);
+		Whiskers u(R"(<ind>using <libraryName> for uint;)");
+		u("ind", "\t");
+		u("libraryName", get<0>(testTuple));
+		usingLibDecl = u.render();
+		Whiskers test(R"(<endl><ind><varDecl><endl><ind><ifStmt>)");
```

Comment:
I'm not sure these placeholders `<endl>`, `<ind>` etc are better than before

Reply:
Hahaha... okay. I'm not sure either. This makes code better formatted than before at the expense of Whisker noise.

Reply:
Haha ok, if you're fine with it I'm fine too

Reply:
yeah, I'm okay with it for now. I can't think of a better solution at the moment. In any case, I can make a stylistic PR if there is an elegant solution at a later point.